Framework/Geometry/inc/MantidGeometry/Instrument/Goniometer.h

```diff
@@ -91,6 +91,10 @@ class MANTID_GEOMETRY_DLL Goniometer {
   // Set rotation angle for an axis in the units the angle is set (default --
   // degrees)
   void setRotationAngle(size_t axisnumber, double value);
+  // Calculate goniometer for rotation around y-axis for constant wavelength
+  // from Q Sample
+  void calcFromQSampleAndWavelength(const Mantid::Kernel::V3D &Q,
```

Comment:
If converted to momentum, this needs to be renamed

Reply:
Using wavelength is better for most of my use cases.